lib/containers/container_images.pm

```diff
@@ -118,14 +118,17 @@ sub test_opensuse_based_image {
     $version = 'Tumbleweed' if ($version =~ /^Staging:/);
 
     if ($image_id =~ 'sles') {
-        my $pretty_version = $version =~ s/-SP/ SP/r;
-        my $betaversion    = get_var('BETA') ? '\s\([^)]+\)' : '';
-        record_info "Validating", "Validating That $image has $pretty_version on /etc/os-release";
-        validate_script_output("$runtime container run --entrypoint '/bin/bash' --rm $image -c 'grep PRETTY_NAME /etc/os-release' | cut -d= -f2",
-            sub { /"SUSE Linux Enterprise Server ${pretty_version}${betaversion}"/ });
+        # SUSE MicroOS does not follow SP versioning
+        unless (is_microos('suse')) {
```

Comment:
Here we're checking the container image but `is_microos` is checking the host. Is this really right?

Reply:
This function checks that the container version is the same as the host, checking for same string `"SUSE Linux Enterprise Server 15 SPX`. For MicroOS testing SLE containers it doesn't make sense since the host will never show `SUSE Linux Enterprise Server 1...` but `SUSE MicroOS ...`. That's why I skip that check for MicroOS case.

Anyway, I haven't enabled the test case eventually. Maybe I can remove this part of the code or just wait for feedback from Release Manager.

Reply:
see the difference between host and container:
http://fromm.arch.suse.de/tests/18#step/podman_image/97
http://fromm.arch.suse.de/tests/18#step/podman_image/98

Reply:
But the `is_microos()` checks the host and the `validate_script_output()` inside the condition checks the container. Why do we check the container only if host isn't MicroOS?

Reply:
Because the script output validation will always fail in MicroOS case, as we have `SUSE Linux Enterprise Server 15 SP2` in the image and something completely different in the host.

Reply:
I've removed this part as I have disabled SLE image tests.